### ISSUE TITLE
[Request New Content] Create a feedback loop for ML in production

### ISSUE THREAD

**Issue by theodu** [kili-technology/kili-blogger-program#3]
### My Request

Create a feedback loop for ML in production

### Community vote
👍 Upvote if you like this topic
🚀 If you can make the article!

Thank you.

**Comment by FnSK4R17s** [comment]
## Summary
The performance of any ML model degrades over time. We always need to label and train our model on new data, especially when we are creating a model that is personalized. Recommender systems and predictive text keyboards are a few examples of models that can benefit from a feedback loop. Feedback loos are easy to implement but they can be frustratingly tough to maintain as feedback loops can often amplify their own biases. Deploying feedback loops without a human in the loop is very risky. However, there are many ways of deploying feedback loops safely in deployment.

## Outline
- Implicit and Explicit Feedback
- User privacy while collecting implicit feedback
- Monitoring of feedback loops efficiently
- Special Cases: Data Flywheel
- Worst case scenario in feedback loops
- User feedback handling
- Implementing a feedback loop using Streamlit and PyTorch on an image classification task

**Comment by FnSK4R17s** [comment]
Hi @theodu,
I agree that there is really small scope for human annotation in an automated feedback loop. However, I have written a tutorial outline just in case. I'd like to suggest that we look into something which is similar to data flywheel as there is a really large scope of using Kili Technology in that use case. I'll create an article proposal for that soon. Let me know if that is something that we should work on. Thanks 

## Introduction
The performance of any ML model degrades over time. We always need to label and train our model on new data, especially when we are creating a model that is personalized. Recommender systems and predictive text keyboards are a few examples of models that can benefit from a feedback loop. Feedback loos are easy to implement but they can be frustratingly tough to maintain as feedback loops can often amplify their own biases. Deploying feedback loops without a human in the loop is very risky. However, there are many ways of deploying feedback loops safely in deployment. In this tutorial, we will be building a feedback loop for an Image Classification Model deployed in production.

## Goals
- To implement a feedback loop using Streamlit, PyTorch and Kili Technology
- To set up efficient monitoring to make sure the model does not reinforce any biases

## Steps
- Step 1: Download the initial training dataset
- Step 2: Train the initial model
- Step 3: Deploy the model using streamlit
- Step 4: Collect data and feedback using the deployed model (along with a brief explanation of concepts like Implicit and Explicit Feedback)
- Step 5: Filter through the collected data and retrain the model
- Step 6: Monitor your feedback loop
